decorators/readme.md

```diff
@@ -0,0 +1,28 @@
+# Decorators in Python
+
+Decorators in Python are a powerful tool that allow us to modify the behavior of functions or methods. They are defined using a special syntax that involves nested functions.
+
+In this example, we define a decorator called 'performance' that measures the execution time of a function. The decorator is defined as a function that takes another function as an argument. Inside the decorator, we define a wrapper function that wraps the original function with additional behavior (in this case, measuring the execution time).
```

Comment:
before providing an example to explain the theory a little bit more for beginners so that it would be easy for them to catch and also provide the syntax with the explanation.